### DISCUSSION TITLE
When will Windows agent support go into GA?

### DISCUSSION THREAD

**Discussion by pbering** [rancher/rke2#4695] [Q&A] [answered]
Hi there, 

When Windows agent support will be GA?

Several places on <https://docs.rke2.io> it is noted that "_Windows Support is currently Experimental as of v1.21.3+rke2r1_" so I assume that as of v1.27.4+rke2r1 this is still the case?

**Comment by brandond** [comment, marked as answer]
The docs need an update. You can find the support matrix at https://www.suse.com/suse-rke2/support-matrix/all-supported-versions/rke2-v1-27/#:~:text=Windows%20Worker%20Node%20Support - see the sidebar for prior minors.

**Comment by pbering** [comment]
Thank you for clarifying!
